tests/upgrades/test_client.py

```diff
@@ -18,139 +18,130 @@
 """
 import time
 
+import pytest
 from fabric.api import execute
 from nailgun import entities
 from upgrade.helpers.docker import docker_execute_command
 from upgrade_tests import post_upgrade
 from upgrade_tests import pre_upgrade
-from upgrade_tests.helpers.constants import FAKE_REPO_ZOO3
 from upgrade_tests.helpers.scenarios import create_dict
 from upgrade_tests.helpers.scenarios import dockerize
 from upgrade_tests.helpers.scenarios import get_entity_data
 
-from robottelo.api.utils import attach_custom_product_subscription
-from robottelo.api.utils import call_entity_method_with_timeout
-from robottelo.constants import DEFAULT_ORG
-from robottelo.test import APITestCase
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.constants import DEFAULT_ARCHITECTURE
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import ENVIRONMENT
+from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_NAME
+from robottelo.constants import PRDS
+from robottelo.constants import REPOS
+from robottelo.constants import REPOSET
+from robottelo.constants.repos import FAKE_0_YUM_REPO
+from robottelo.constants.repos import FAKE_9_YUM_REPO
 from robottelo.test import settings
 
+# host machine for containers
+docker_vm = settings.upgrade.docker_vm
+DOCKER_VM = settings.upgrade.docker_vm
```

Comment:
Was one of these supposed to be cleaned up?

Reply:
In `robottelo.properties` its in lower case, but confusingly in the `dockerize` function, in `upgrade_tests/helpers`, its in upper case. Also in the new dynconf its in upper case. So only after further testing will this be removed.

Reply:
since this is a variable local to this file, you don't need to consider any of that. as long as you use the caps version of the constant in this file, then you're good.